app/views/discovery_rules/edit.html.erb

```diff
@@ -1,3 +1,3 @@
-<% title _("Edit Discovery Rule") %>
+<% title _('Edit %s' % @discovery_rule.name) %>
```

Comment:
Must be `_('Edit %s') % @discovery_rule.name`

Reply:
@sharvit, you'ved made a format string, but called the format content inside the underscore function.

What @lzap wrote, is that you should do it on the result of the underscore function, and not as part of it's original string.

Reply:
updated